code/modules/clothing/spacesuits/spacesuits.dm

```diff
@@ -7,7 +7,6 @@
 	icon_state = "space"
 	desc = "A special helmet designed for work in a hazardous, low-pressure environment."
 	item_flags = STOPPRESSUREDAMAGE | THICKMATERIAL | AIRTIGHT
-	flags_inv = BLOCKHAIR
```

Comment:
Intentional?

Reply:
Yes, there are two flags inv in the same object.